### PR TITLE
Track user count

### LINKED ISSUE #540: Count user number
When app is installed and uninstalled so we can track usage
going with https://matomo.org/

### PR DESCRIPTION
**Pull request by BeroBurny**
**Short description of work done**

- implement user count tracking
- refactor consent screen

### PR Checklist

- [x] I have run type check locally
- [x] I have run linter locally
- [ ] I have run unit and integration tests locally
- [x] Rebased to master branch / merged master

### Changes

- implement matomo tracker sdk
- refactor consent
- implement tracking

### Example

Screenshot from 2021-03-31 10-22-48

### Issues

Closes #540

### CODE REVIEW

File: src/renderer/containers/Consent/Consent.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        history.push(Routes.DASHBOARD_ROUTE);
+    };
+
+    return (
+        <Background>
+            <Modal>
+                <h1>Collecting data Consent?</h1>
+                <p>
+                    By sharing ChainGuardian bug reports and analytics data we will be able to improve your app
+                    experience. Don&#39;t worry! We will never be able to access your keys or share to 3th party.
```
Comment (mpetrunic) [inline src/renderer/containers/Consent/Consent.tsx:29]:
```suggestion
                    experience. Don&#39;t worry! We will never be able to access your keys or share to 3rd party.
```

File: src/renderer/ducks/settings/sagas.ts [hunk 2]
```diff
@@ -14,7 +15,12 @@ function* loadAccountSettings(): Generator<PutEffect | Promise<Settings | null>,
 function* saveAccountSettings(action: {payload: ISettings}): Generator<PutEffect | Promise<void>> {
     yield database.settings.set(DEFAULT_ACCOUNT, action.payload);
 
-    if (action.payload.reporting !== undefined) yield put(setReporting(action.payload.reporting));
+    if (action.payload.reporting !== undefined) {
```
Comment (mpetrunic) [inline src/renderer/ducks/settings/sagas.ts:18]: Wouldn't this prevent changing reporting from true to false or reverse?

File: src/renderer/containers/Consent/Consent.tsx (hunk 1 above)
Comment (BeroBurny) [reply src/renderer/containers/Consent/Consent.tsx:29]: text will be changed after @morrigan  provide it :sweat_smile:

File: src/renderer/ducks/settings/sagas.ts (hunk 2 above)
Comment (BeroBurny) [reply src/renderer/ducks/settings/sagas.ts:18]: `ISettings` all params are optional and this cover a case if is reporting not provided

**Pull request merged**